spec/views/submissions/new.html.erb_spec.rb

```diff
@@ -3,7 +3,6 @@
 RSpec.describe 'submissions/new', type: :view do
   before do
     assign(:submission, Submission.new(
-                          url: 'MyText',
```

Comment:
The `Submission` model doesn't have a `url` field on it, so this was throwing an error.